azure-keyvault/azure/keyvault/aio/secrets/_client.py

```diff
@@ -164,22 +224,27 @@ def vault_url(self):
             raise
 
     async def list_secret_versions(self, name: str, **kwargs: Mapping[str, Any]) -> AsyncGenerator[SecretAttributes, None]:
+        # type: (str, Mapping[str, Any]) -> Generator[SecretAttributes]
         """List all versions of the specified secret.
 
         The full secret identifier and attributes are provided in the response.
-        No values are returned for the secrets. This operations requires the
+        No values are returned for the secrets. This operation requires the
         secrets/list permission.
 
-        :param name: The name of the secret.
-        :type name: str
-        :param max_page_size: Maximum number of results to return in a page. If
-         not max_page_size, the service will return up to 25 results.
-        :type maxresults: int
-        :return: An iterator like instance of Secret
+        :param str name: The name of the secret.
+        :returns: An iterator like instance of Secret
         :rtype:
-         ~azure.keyvault.secrets.SecretAttributesPaged[~azure.keyvault.secret.Secret]
+         ~azure.keyvault.secrets._models.SecretAttributesPaged[~azure.keyvault.secrets._models.SecretAttributes]
```

Comment:
I don't know what the correct rst would be here, but it would be something else because this returns an iterable of `SecretAttributes`.  🤔